pandas/tseries/index.py

```diff
@@ -64,6 +64,7 @@ def f(self):
             if self.tz is not utc:
                 values = self._local_timestamps()
 
+        # boolean accessors -> return array
```

Comment:
I think it might be worth it to add something like this:

```
class DatetimeIndex....:


    _boolean_ops = ['is_month_start'......]
    _datetimelike_ops = [....] + _boolean_ops
```

then you can use that here.